src/Process/3dTilesProcessing.js

```diff
@@ -79,7 +80,74 @@ export function $3dTilesCulling(node, camera) {
     return false;
 }
 
+// Cleanup all 3dtiles|three.js starting from a given node n.
+// n's children can be of 2 types:
+//   - have a 'content' attribute -> it's a tileset and must
+//     be cleaned with cleanup3dTileset()
+//   - doesn't have 'content' -> it's a raw Object3D object,
+//     and must be cleaned with _cleanupObject3D()
+function cleanup3dTileset(layer, n, depth = 0) {
+    // If this layer is not using additive refinement, we can only
+    // clean a tile if all its neighbours are cleaned as well because
+    // a tile can only be in 2 states:
+    //   - displayed and no children displayed
+    //   - hidden and all of its children displayed
+    // So here we implement a conservative measure: if T is cleanable
+    // we actually only clean its children tiles.
+    const canCleanCompletely = layer.additiveRefinement || depth > 0;
```

Comment:
Additive refinement is handled at the tile level. You can have different type of refinement in the same layer.

```js
n.additiveRefinement
```

Reply:
Thanks fixed.